assets/js/base/components/chip/stories/removable-chip.stories.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import { Story, Meta } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import { RemovableChip, RemovableChipProps } from '../removable-chip';
+
+export default {
+	title: 'WooCommerce Blocks/@base-components/RemovableChip',
+	component: RemovableChip,
+} as Meta< RemovableChipProps >;
+
+const Template: Story< RemovableChipProps > = ( args ) => (
+	<RemovableChip { ...args } />
+);
+
+export const Default = Template.bind( {} );
+Default.args = {
+	text: 'Take me to the casino',
+	screenReaderText: "I'm a removable chip, me",
+	onRemove: () => {
+		window.alert( 'Chip Removed!' ); //eslint-disable-line
```

Comment:
There shouldn't be any need for this. See: https://storybook.js.org/docs/react/essentials/actions/

I'm not sure how removing the chip actually works, but if it's easy to mock and it has some interesting features (like animations and whatnot) it'd be nice to have it in the Storybook. Otherwise as long as it appears in the Actions panel, that's basically the purpose of it, to avoid those alerts or console logs.

Reply:
Aha nice! It looks like to remove the chip it just relies on the `onRemove` function that it takes as a prop. So nothing is hardcoded in, it's up to the consumer to define how the chip is removed.